lib/saml_idp/xml_security.rb

```diff
@@ -298,14 +254,10 @@ def digests_match?(hash, digest_value)
       end
 
       def extract_signed_element_id
-        reference_element = REXML::XPath.first(
-          self,
-          '//ds:Signature/ds:SignedInfo/ds:Reference',
-          { 'ds' => DSIG }
-        )
+        reference_element = document.at_xpath('//*:Signature/*:SignedInfo/*:Reference')
         return if reference_element.nil?
 
-        self.signed_element_id = reference_element.attribute('URI').value[1..-1]
+        @signed_element_id = reference_element.attribute('URI').value[1..-1]
```

Comment:
Where is this used? Can we remove the method entirely given the change to `initialize`?

Reply:
yes, i think you're right!

Reply:
updated in https://github.com/18F/saml_idp/pull/110/commits/54d2d1565f7ce201ca494272df69bd6ab4b9cc5f